src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/BotSequence.cs

```diff
@@ -36,6 +36,11 @@ public class BotSequence
          */
         public string description;
 
+        /**
+         * <summary>LastModified is the date that the sequence file was last altered</summary>
+         */
+        public DateTime lastModified;
```

Comment:
Do we need to update a version field for this change? I just noticed that I don't see any api version field on this data type... I wonder if there is a reason for that. We can keep this as a note to ask when Zack gets back I suppose.

Reply:
I think we're ok without an `apiVersion` for now. This `BotSequence` file doesn't have any consumers other than this brand new UI work I'm doing